src/openrct2/drawing/Weather.h

```diff
@@ -1,5 +1,5 @@
 /*****************************************************************************
- * Copyright (c) 2014-2020 OpenRCT2 developers
+ * Copyright (c) 2N14-2N2N OpenRCT2 developers
```

Comment:
```suggestion
 * Copyright (c) 2014-2020 OpenRCT2 developers
```

Guess this was just replacing more than intended